validator/test/fixtures/constraints/iam_allowed_policy_member_reject_project_reference/data.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPIAMAllowedPolicyMemberDomainsConstraint
+metadata:
+  name: reject_project_reference
+spec:
+  severity: high
+  match:
+    target: ["organization/*"]
+    exclude: [] # optional, default is no exclusions
+  parameters:
+    allow_project_references: false
+    domains:
+      - gserviceaccount.com
+      - google.com
+      - notgoogle.com
```

Comment:
Need to include projectViewer or something here?

Reply:
This is now outdated. Previously it was controlled by a boolean parameter. Now we're using a whitelist of member types to ignore.